server/app.js

```diff
@@ -1,11 +1,21 @@
-import express from 'express';
-import http from 'http';
+const express = require('express');
+const mongoose = require('mongoose');
+const bodyParser = require('body-parser');
+const routes = require('./routes/news_routes');
+const app = express();
 
-var app = express();
+mongoose.Promise = global.Promise;
```

Comment:
Hi, so what is the purpose of the mongoose.Promise?

Reply:
This line tells mongoose to use JavaScript default Promise.